content/pl/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
+title: Kubernetes — co to jest?
+content_template: templates/concept
+weight: 10
+card:
+  name: concepts
+  weight: 10
+---
+
+{{% capture overview %}}
+Na tej stronie znajdziesz ogólne informacje o Kubernetesie.
```

Comment:
Is `Kubernetesie` in this sentence correct?

Reply:
Yes, it is.

Reply:
As per discussion on the Slack channel, we stick to general Polish grammar rules for inflection of names like "Kubernetes".